### ISSUE TITLE
Menu position does not work correctly

### ISSUE THREAD

**Issue by rr8733380** [notDmDrl/pull_down_button#21]
The menu is not shown when the widget has a large height, greater than the screen but the opacity appears

My Code: 
```
return PullDownButton(
  itemBuilder: (context) {
    return [...items];
  },
  buttonBuilder: (context, showMenu) {
    return GestureDetector(
      onTap: () async {
        showMenu();
      },
      child: BigHeightWidget,
    );
  },
  position: PullDownMenuPosition.automatic,
);
```

**Comment by notDmDrl** [comment]
Hi @rr8733380 

For the cases like this i would strongly advise using either `PullDownMenuPosition.over` or `showPullDownMenu` function with desired position provided.

`PullDownMenuPosition.automatic` (or any other for that matter) will always be forced to be above or under `buttonBuilder`, which with your example will not work at all.

**Comment by rr8733380** [comment]
thanks for the help and quick response boss

**Comment by notDmDrl** [comment]
Should be fixed with **0.8.1** update.
